cli/index.ts

```diff
@@ -133,11 +133,11 @@ const warehouseOption: INamedOption<yargs.PositionalOptions> = {
   }
 };
 
-const verboseOutputOption: INamedOption<yargs.Options> = {
-  name: "verbose",
+const jsonOutputOption: INamedOption<yargs.Options> = {
+  name: "json",
   option: {
     describe:
-      "If true, the full contents of command output will be output (containing fully compiled SQL, etc).",
+      "Exclusively output a JSON describing the non-executable graph (other output is suppressed, excluding errors).",
```

Comment:
"Outputs a JSON representation of the compiled project" - I think is slightly more clear.

Reply:
Ah nice, I guess they'll see what happens anyway when they run it